swap_meet/item.py

```diff
@@ -1,2 +1,26 @@
+import uuid
+
 class Item:
-    pass
\ No newline at end of file
+    def __init__(self, id=None, condition=0):
+        id = int(uuid.uuid4()) if not id else id
+        self.id = id
+        self.condition = condition
+
+    def get_category(self):
+        return "Item"
+    
+    def __str__(self):
+        return f"An object of type Item with id {self.id}."
+    
+    def condition_description(self):
+        conditional_phrases = [
+            "yikes", "not worth it", 
+            "it could work", 
+            "just a scratch",
+            "almost new", 
+            "as good as it gets"
+            ]
+        return conditional_phrases[self.condition]
```

Comment:
This could also be a time to use match case! 
```python
    def condition_description(self):
        match self.condition:
            case 0:
                return "Throw Away"
            case 1:
                return "Eh"
            case 2:
                return "Better..."
            case 3:
                return "..."
            case 4: 
                return "Okay?!"
            case 5:
                return "Yes!"
```